cron/cronfile.js

```diff
@@ -0,0 +1,25 @@
+const newSchema = require('../models/schema_DB');
+const cron = require("node-cron");
+btoa = require('btoa');
+require('dotenv').config();
+
+module.exports = (app) => {
+    cron.schedule("0 6 * * *", function() {
+        console.log("running daily at 6");
+        for(var i = 0; i < process.env.TIMES;i++){
+            var d = new Date();
+            const code = d.getTime(); 
+            console.log(code);
```

Comment:
no console.log allowed in prod systems , so remove it and still if you want to test in production 
use logger  properly 
https://blog.risingstack.com/node-js-logging-tutorial/